pkg/proxy/plugin.go

```diff
@@ -0,0 +1,126 @@
+package proxy
+
+import (
+	"context"
+	"github.com/matrixorigin/matrixone/pkg/common/moerr"
+	"github.com/matrixorigin/matrixone/pkg/common/morpc"
+	"github.com/matrixorigin/matrixone/pkg/logutil"
+	"github.com/matrixorigin/matrixone/pkg/pb/plugin"
```

Comment:
please seperate internal packages and external packages with a blank line, and add copyright header.